invenio_rdm_migrator/transform/identity.py

```diff
@@ -18,33 +18,3 @@ class IdentityTransform(Transform):
     def _transform(self, entry):
         """Transform entry."""
         return entry
-
-
-class IdentityDictKeyMixin:
```

Comment:
It was creating false positives. Mainly for updates. For example, if the payload brought `{a: 1234}` but the actual transform required `_a` and `_b` methods, it was creating a resulting `{a: 1234, b: None}` when in reality we want to ignore `b`. 

The desired behavior can be achieved with the `IdentityTransform` , as in the `OAuthRemoteToken` example in this PR.